plugins/module_utils/foreman_helper.py

```diff
@@ -575,6 +575,21 @@ def _patch_organization_ignore_types_api(self):
             _organization_create['params'].append(_ignore_types_param)
             _organization_update['params'].append(_ignore_types_param)
 
+    @_check_patch_needed(fixed_version='3.8.0', plugins=['katello'])
+    def _patch_products_repositories_allow_nil_credential(self):
+        """
+        This is a workaround for the missing allow_nil: true in the Products and Repositories controllers
```

Comment:
This is a in two-three releases of Katello this work around gets dropped?

Reply:
We've been rather conservative in dropping those apidoc patches, but the code is a noop once it detects a new enough version.